omise-woocommerce.php

```diff
@@ -81,6 +81,7 @@ public function init() {
 		register_omise_creditcard();
```

Comment:
Feels like we should have a list of payment methods, then iterate over a list to call these `register_omise_xxxx` functions. Such a list would also be beneficial for including the files etc. - when a new payment method was added, you could simply add a new name to the list